kernel/kernel-api/src/test/scala/io/delta/kernel/internal/snapshot/LogSegmentSuite.scala

```diff
@@ -346,4 +348,112 @@ class LogSegmentSuite extends AnyFunSuite with MockFileSystemClientUtils {
     assert(logSegment.toString === expectedToString)
   }
 
+  def deltaFileStatus(v: Long): FileStatus =
```

Comment:
nit: can you add this to `MockFileSystemClientUtils` ?

And refactor `deltaFileStatuses` to use `deltaFileStatus` etc?